src/raftstore/store/peer_storage.rs

```diff
@@ -1356,6 +1365,25 @@ pub fn write_peer_state<T: Mutable>(
     Ok(())
 }
 
+pub fn write_merge_state<T: Mutable>(
```

Comment:
Does it conflict with `write_peer_state`? `write_peer_state` will clear `merge_state`.

Reply:
Yes. But it won't be called unless merge is finished or rolled back.